pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3005,7 +3008,8 @@
       ],
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.0.0.0": "4.5.0"
+        "4.0.0.0": "4.5.1",
```

Comment:
The assembly version's package version should be the first where the assembly was shipped. In this case 4.5.0 and not 4.5.1. Same for other below.

Reply:
No it should be the latest package that shipped stable. We want to ensure that folks are getting the latest version of any changes with the same assembly version.

Reply:
hmm makes sense but @ericstj told me the opposite in a random PR which I can't find right now.